modules/ROOT/pages/design-import-files-api-fragment-projects.adoc

```diff
@@ -0,0 +1,58 @@
+= Import Files into an API-Fragment Project
+
+You can import RAML fragments into an API-fragment project. The fragments that you import can be written in RAML 0.8 or RAML 1.0. You can import annotation types, data types, examples of JSON response payloads, libraries, resource types, security schemes, and traits. You must set one of the fragments in your project as the root file of the project.
```

Comment:
Should we tell WHY?

Reply:
@kbostrom , users will know why they want to import files.